cfgov/v1/atomic_elements/molecules.py

```diff
@@ -246,6 +246,25 @@ class Meta:
         label = 'Phone'
 
 
+class ImageInset(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False)
+    paragraph = blocks.RichTextBlock(required=False)
+    image = atoms.ImageBasic()
+    is_image_left = blocks.BooleanBlock(required=False,
+                                        label='Image float left')
+    image_width = blocks.ChoiceBlock(choices=ref.inset_image_widths,
```

Comment:
Could this be changed to have `choices` refer to a callable, so that changing the options in future won't require a migration? Something like:

```py
# in ref.py
def get_inset_image_widths():
    return inset_image_widths

# in molecules.py
class ImageInset(blocks.StructBlock):
    ...
    image_width = blocks.ChoiceBlock(choices=ref.get_image_inset_widths, ...)
```